README.md

```diff
@@ -1 +1,292 @@
+     memoryUsage: [Function: memoryUsage],
+     binding: [Function: binding],
+     _linkedBinding: [Function: _linkedBinding],
+     _events:
+      { newListener: [Function],
+        removeListener: [Function],
+        SIGWINCH: [Object] },
+     _rawDebug: [Function],
+     domain: [Getter/Setter],
+     _maxListeners: undefined,
+     EventEmitter:
+      { [Function: EventEmitter]
+        EventEmitter: [Circular],
+        usingDomains: true,
+        defaultMaxListeners: 10,
+        init: [Function],
+        listenerCount: [Function] },
+     _fatalException: [Function],
+     _exiting: false,
+     assert: [Function],
+     config: { target_defaults: [Object], variables: [Object] },
+     nextTick: [Function: nextTick],
+     _tickCallback: [Function: _tickDomainCallback],
+     _tickDomainCallback: [Function: _tickDomainCallback],
+     stdout: [Getter],
+     stderr: [Getter],
+     stdin: [Getter],
+     openStdin: [Function],
+     exit: [Function],
+     kill: [Function],
+     _eventsCount: NaN },
+  GLOBAL: [Circular],
+  root: [Circular],
+  Buffer:
+   { [Function: Buffer]
+     poolSize: 8192,
+     isBuffer: [Function: isBuffer],
+     compare: [Function: compare],
+     isEncoding: [Function],
+     concat: [Function],
+     byteLength: [Function: byteLength] },
+  clearImmediate: [Function],
+  clearInterval: [Function],
+  clearTimeout: [Function],
+  setImmediate: [Function],
+  setInterval: [Function],
+  setTimeout: [Function],
+  console: [Getter],
+  module:
+   Module {
+     id: 'repl',
+     exports:
+      { writer: [Object],
+        _builtinLibs: [Object],
+        REPLServer: [Object],
+        REPL_MODE_SLOPPY: Symbol(repl-sloppy),
+        REPL_MODE_STRICT: Symbol(repl-strict),
+        REPL_MODE_MAGIC: Symbol(repl-magic),
+        start: [Function],
+        repl: [Object] },
+     parent: undefined,
+     filename: '/Users/azat/Documents/Code/learn-co/node-debugger/repl',
+     loaded: false,
+     children: [],
+     paths:
+      [ '/Users/azat/Documents/Code/learn-co/node-debugger/repl/node_modules',
+        '/Users/azat/Documents/Code/learn-co/node-debugger/node_modules',
+        '/Users/azat/Documents/Code/learn-co/node_modules',
+        '/Users/azat/Documents/Code/node_modules',
+        '/Users/azat/Documents/node_modules',
+        '/Users/azat/node_modules',
+        '/Users/node_modules',
+        '/node_modules' ] },
+  require:
+   { [Function: require]
+     resolve: [Function],
+     main: undefined,
+     extensions: { '.js': [Function], '.json': [Function], '.node': [Function] },
+     cache: {} },
+  _: [Circular],
+```
+
+Let's cover the most important of them in more details.
+
+Note: Another name for `global` is `GLOBAL`. They are aliases. So `global.user` and `GLOBAL.user` are the same.
+
+## Global Object Properties
+
+So what are the Node-specific global interfaces? Here's the list of Node global interfaces which do not exist in browser JavaScript:
```

Comment:
What does "Node-specific global interface" mean? I don't know how to compare them to browser js because I don't know what the means